compiler/AST/symbol.cpp

```diff
@@ -1768,6 +1770,15 @@ void FnSymbol::finalizeCopy(void) {
      */
     update_symbols(this, map);
 
+    // Replace vararg formal if appropriate.
+    if (this->varargOldFormal) {
+      substituteVarargTupleRefs(this->body, this->varargNewFormals.size(),
```

Comment:
I do not know this rule. We can assume that diverse formatting styles are acceptable.